src/module/item/spellcasting-entry/document.ts

```diff
@@ -93,7 +93,7 @@ class SpellcastingEntryPF2e<TParent extends ActorPF2e | null = ActorPF2e | null>
 
     override prepareBaseData(): void {
         super.prepareBaseData();
-        this.system.proficiency.slug ||= this.system.tradition.value;
+        this.system.proficiency.slug ||= "spellcasting-base";
```

Comment:
Why base? There's only ever one spellcasting proficiency

Reply:
The main thing I struggled with is that if I went spellcasting, the call would be actor.getSpellcasting("spellcasting"), which should probably return the "highest" spellcasting instead (does nothing right now).

I considered doing something like `actor.getSpellcasting("spellcasting, { type: "base" })` but since this is all prepared data anyways (spellcasting-base isn't persisted), I wanted to defer the question until spellcasting could be automated.

Still not quite sure what to do about that entire ordeal.